setup.py

```diff
@@ -0,0 +1,25 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="",
+    version="0.0.1",
+    author="InfraCode",
+    author_email="",
+    description="InfraRapid package",
+    long_description=long_description,
```

Comment:
Let's not take readme file, since it is going to grow a lot. Just:
"Infrarapid tools allows you to create, configure and deploy same infrastructure over multiple cloud providers"